reference/docs/conversion/README.md

```diff
@@ -6,4 +6,19 @@ You should be able to check just this directory out with subversion using the co
 
 Or just clone it in github.
 
-You need to run all scripts in order to ensure a proper conversion. For instance if upgrading to 11, if you're on 10.3 you need to see if scripts exists for 10.4, 10.5, 10.6, etc before running the 11 script. See this guide for more information: https://confluence.sakaiproject.org/display/DOC/Sakai+upgrade+information
+You need to run all scripts in order to ensure a proper conversion. For instance if upgrading to 11, if you're on 10.3 you need to see if scripts exists for 10.4, 10.5, 10.6, etc before running the 11 script. 
```

Comment:
if scripts exists => scripts exist

before running the 11 script => 11.0 script